projects/demo/src/modules/app/home/home.module.ts

```diff
@@ -13,6 +14,9 @@ import {HomeComponent} from './home.component';
         TuiNotificationModule,
         TuiLinkModule,
         TuiAccordionModule,
+        TuiTabsModule,
+        CommonModule,
+        TuiSvgModule,
```

Comment:
Let's not use SVG going forward. If you have a colored SVG - just use IMG, if you have a single color icon - let's use `tui-icon` instead.

Reply:
fixed